### ISSUE TITLE
Photovoltaic Relay Library

### ISSUE THREAD

**Issue by tomhajjar** [ra3xdh/qucs_s#222]
First cut at creating a Photovoltaic Relay for the Opto Isolator Library. The CPC1018N model from the LTspice Forum is more sophisticated than the very simple factory supplied Vishay VOR14642.

I modified IXYS model CPC1018N_R05.cir and created CPC1018N_sub.cir. It isn't working so either I did something wrong or it needs to be modified. 

![2023-02-15_204101](https://user-images.githubusercontent.com/102838604/219244222-bd95ee0b-6ba8-441d-ba69-bb2dae09ded9.jpg)

![2023-02-15_204115](https://user-images.githubusercontent.com/102838604/219244241-eb66525d-d425-418b-8a06-b8ec279bf54f.jpg)

[Photovoltaic_Relays_prj.zip](https://github.com/ra3xdh/qucs_s/files/10751317/Photovoltaic_Relays_prj.zip)

TODO list: 

- [ ] Provide photovoltaic relays library

**Comment by ra3xdh** [comment]
> I modified IXYS model CPC1018N_R05.cir and created CPC1018N_sub.cir. It isn't working

The CPC1018N model works, but the testbench circuit is wrong. According the datasheet the switching speed of this device is in range 0.22-2ms, but you a using 50us pulse to test the device. 
![image](https://user-images.githubusercontent.com/4920080/219289862-6223eca9-6f60-4acb-842b-a74a986dd562.png)

**Comment by tomhajjar** [comment]
Unfortunately there is limited Photovoltaic Relay models available and most are too simple to be useful if the relay is used in a circuit that switching speed is pertinent. However in most cases Photovoltaic Relays are used as power switches for slow devices where speed or response time is immaterial. I don't like that the Vishay VOR14642 model and others I found don't model the input diode.

The CPC1018N model includes:

.PARAM IFON=1m
.PARAM RON=0.65
.PARAM TOFF=250u
.PARAM COUT=25p
.PARAM CIO=1p

Is there a way to include some of these like Ron in the library so this model can be used for other relays? I realize what I did below caused a nested sub but it works.  

[Op_Amp_Sub_Param_Pass_prj.zip](https://github.com/ra3xdh/qucs_s/files/10756177/Op_Amp_Sub_Param_Pass_prj.zip)
